samples/widgets/feature-layer-class/src/runtime/widget.tsx

```diff
@@ -19,15 +19,16 @@
 */
 import { React, IMDataSourceInfo, DataSource, DataSourceManager, DataSourceStatus, FeatureLayerQueryParams, AllWidgetProps, DataSourceComponent } from 'jimu-core';
 
-interface State {
+interface IState {
```

Comment:
Add an `I` for interface is OK, but we don't add it in other widgets.

Reply:
ok, I will remove it for consistency.

Reply:
removed.